Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -232,6 +264,13 @@ public final class WalletManager: Equatable, CustomStringConvertible {
                    system: system,
                    callbackCoordinator: callbackCoordinator,
                    take: false)
+
+        // Register a wallet for each currency.
+        currencies
+            .forEach {
+                if network.hasCurrency ($0) {
+                    cryptoWalletManagerRegisterWallet (core, $0.core) }
```

Comment:
Does this leak the returned wallet?

Reply:
Yes.  Will call `self.registerWalletFor(currency)` - doing the right thing *and* avoiding duplicate logic.